requirements/testing.txt

```diff
@@ -1,7 +1,7 @@
 -r pip.txt
 -r local-docs-build.txt
 
-django-dynamic-fixture==2.0.0
+django-dynamic-fixture==3.1.0
```

Comment:
Included this for the Travis CI build but can remove.

Reply:
Tests are still failing with the upgrade, we can let the previous version, so we can merge this and do the rest in another PR if you want.

Reply:
It'd be good to fix those tests in this PR but I'm not sure where to start looking.

I've looked at the `setUp()` methods in test_privacy_urls.py and test_core_utils.py but I don't see anything wrong with the way the dynamic models are set.

Reply:
Fixed the test_privacy_urls.py tests 👍

Reply:
Fixed test_core_utils.py 👍

Reply:
We can put back the django-dynamic upgrade here.

Reply:
Won't the upgrade be made by pyup?
https://github.com/readthedocs/readthedocs.org/pull/7012

Reply:
We can do the update here, so we know all tests pass with the new version